### ISSUE TITLE
Issue to track progress on TODO Guide PDF design | LF request

### ISSUE THREAD

**Issue by anajsana** [todogroup/outbound-oss#58]
I've opened a request for the LF Marketing/Design team to start working on the pdf design and landing page on LF website. A draft should be ready by Aug 12 and the final version by Sep 5 👍 

This issue is to keep contributors to this guide in the loop of important updates from the design team 🙂

**Comment by misappi** [comment]
@anajsana: Is the draft version ready? It was planned to be ready by Aug 12. Thanks!

**Comment by anajsana** [comment]
@misappi yes! I'll share the pdf version via email for your review tomorrow morning! Sorry I've been on vacation these days and couldn't give any updates earlier 🥲

**Comment by anajsana** [comment]
Email send! In addition to this, we should decide on 6 or 9 headlines to create the infographic that summarizes the main outcomes of this guide (that will be placed on page Nº2)

**Comment by misappi** [comment]
@anajsana: I added a TL;DR / key takeaways to the README, see #74.
